csv_template_ingester/tests/test_template_csv_parser.py

```diff
@@ -23,6 +23,17 @@ def test_get_keyword():
     assert keyword_two == 'NAME'
     assert column_header_two == ''
 
+def test_get_keyword_colon_count():
+    """
+    Tests the expected ValueError for >2 colons in the header.
+    Test fails if ValueError not thrown
+    """
+    try:
+        keyword, syst, column_header = get_keyword('PROPERTY::: Test name (MPa)')
+    except ValueError:
+        keyword = '>2_colons'
+    else:
+        assert keyword == '>2_colons', "ValueError was not raised"
```

Comment:
If the program gets to this line, it has failed the test no matter what.  To make that more clear, you could just `assert False, "ValueError was not raised"`

Reply:
Yes, that would make a lot more sense to me.